project.godot

```diff
@@ -70,6 +70,7 @@ config/icon="res://Icon.png"
 [autoload]
 
 GoDAW="*res://addons/godaw_toolkit/API/GoDAW.gd"
+Global="*res://util/scripts/Global.gd"
```

Comment:
is global used anywhere now?

Reply:
it was removed in the last commit...

Reply:
Whoops, had an extra autoload script :sweat_smile: